### PR TITLE
Add helm chart

### PR DESCRIPTION
**Pull request by tobytoyuito**

### CODE REVIEW

File: helm-charts/namespace-cleaner/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+appVersion: "1.0"
+description: Delete expired namespace
```
Comment (paulgmiller) [inline helm-charts/namespace-cleaner/Chart.yaml]: put a link back to github readme in the description?

File: helm-charts/namespace-cleaner/values.yaml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    limits:
+      cpu: "1"
+      memory: "1000Mi"
+    requests:
+      cpu: "0.75"
+      memory: "500Mi"
+  # max namespace inactive hours
+  maxAge: "24"
+  restartPolicy: Never
+  activeDeadlineSeconds: 120
```
Comment (paulgmiller) [inline helm-charts/namespace-cleaner/values.yaml]: no whitelist of namepaces in her. Should we just rip it out completely

File: helm-charts/namespace-cleaner/templates/secret.yaml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+kind: Secret
+apiVersion: v1
+metadata:
+  labels:
+    app: {{ template "namespace-cleaner.name" . }}
+    chart: {{ template "namespace-cleaner.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
```
Comment (paulgmiller) [inline helm-charts/namespace-cleaner/templates/secret.yaml]: need to educate my self about release and heritage in helm.

File: helm-charts/namespace-cleaner/templates/_helpers.tpl [hunk 4]
```diff
@@ -0,0 +1,32 @@
+{{/* vim: set filetype=mustache: */}}
```
Comment (paulgmiller) [inline helm-charts/namespace-cleaner/templates/_helpers.tpl]: do we have to mousage every line? Can we not do 
{{ /* Expand the name of hte chart.*/
define "namespace-cleaner.name
default .....
}}

File: helm-charts/namespace-cleaner/templates/_helpers.tpl [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+{{- if .Values.fullnameOverride -}}
+{{- .Values.fullnameOverride | trunc 63 | trimSuffix "-" -}}
+{{- else -}}
+{{- $name := default .Chart.Name .Values.nameOverride -}}
+{{- if contains $name .Release.Name -}}
+{{- .Release.Name | trunc 63 | trimSuffix "-" -}}
+{{- else -}}
+{{- printf "%s-%s" .Release.Name $name | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+{{- end -}}
```
Comment (paulgmiller) [inline helm-charts/namespace-cleaner/templates/_helpers.tpl]: ends map to and if or a define?

File: helm-charts/namespace-cleaner/templates/cronjob.yaml [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+      template:
+        spec:
+          serviceAccountName: {{ template "namespace-cleaner.fullname" . }}
+          containers:
+          - name: cleaner
+            image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+            imagePullPolicy: {{ .Values.image.pullPolicy }}
+            resources:
+              {{- toYaml .Values.cleaner.resources | nindent 14 }}
+            env:
```
Comment (paulgmiller) [inline helm-charts/namespace-cleaner/templates/cronjob.yaml]: NS_WHITELIST?

File: helm-charts/namespace-cleaner/Chart.yaml (hunk 1 above)
Comment (tobytoyuito) [reply helm-charts/namespace-cleaner/Chart.yaml]: added home and sources to this yaml. They are for links

File: helm-charts/namespace-cleaner/templates/_helpers.tpl (hunk 5 above)
Comment (tobytoyuito) [reply helm-charts/namespace-cleaner/templates/_helpers.tpl]: yes. it ends 2 if and 1 define. Added indentation for readability

**Pull request merged**